recipe/build.sh

```diff
@@ -21,4 +21,5 @@ fi
 ./configure --with-pic --prefix=$PREFIX --disable-dependency-tracking
 make
 make check
+ln README.md README  # hack; snappy autotools build is broken in 1.1.6
```

Comment:
Yeah, though it seems like that doesn't actually fix it since it makes `README.tmp` instead of the expected `README`? The build fails when it tries to install `README` (see build log for the first commit) but works after doing this. I supposed we could patch the build system instead, I just haven't looked into that yet.